sencha-workspace/SlateDemonstrationsTeacher/app/controller/Viewport.js

```diff
@@ -6,16 +6,16 @@ Ext.define('SlateDemonstrationsTeacher.controller.Viewport', {
 
 
     views: [
-        'Dashboard@Slate.cbl.view.teacher'
+        'Dashboard@SlateDemonstrationsTeacher.view.main'
```

Comment:
@aliyeysides now that this class is within the app you could just use `'main.Dashboard'`. "main" things should just be in the top-level of the views namespace though that extra folder doesn't contribute any organization so in the end it should just be `views: [ 'Dashboard' ]`